src/fe_utils/log.c

```diff
@@ -100,10 +100,8 @@ cbdb_log(cbdb_log_level level, const char* file, int line, const char* format, .
     len = snprintf(NULL, 0, fmt, timestamp, s_level[level], file, line);
```

Comment:
the `len` looks is not used any more , can it be removed ?

Reply:
Yes, because I was uncertain whether `len` would always be greater than 0, I was afraid unnecessary `,` might be written to the log file. Maybe we can use the return values of fprintf and vfprintf to achieve this?
```
len = fprintf(log_file, fmt, timestamp, s_level[level], file, line);
if (len> 0)
{
    fprintf(log_file, ",");
}

// record log information
va_list arg_ptr;
va_start(arg_ptr, format);
len = vfprintf(log_file, format, arg_ptr);
va_end(arg_ptr);

if (len > 0)
{
    fprintf(log_file, "\n");
}
```

Reply:
I think the len woundn't be 0, because `fmt` is not empty.

Reply:
oh...yes. it can be removed. I will update the code.